code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -33,9 +33,10 @@
 			if(M.confused < drunk_value)
 				M.confused += 3
 		if(volume >= boozepwr*3.8)
-			M.adjustToxLoss(1)
-	..()
-	return
+			M.adjustToxLoss(1, 0)
+			. = 1
+	return ..() || .
```

Comment:
@phil235 

I know this is ultra late, but I didn't notice this at the time. What particularly reason did you have for doing `return ..() || .`?

Why not just have `. = 1` somewhere and then just `..()` as per normal. What value is even being returned? (this may apply more to the updated versions of this than the current, in this PR).